ui/components/app/collectible-details/collectible-details.js

```diff
@@ -101,34 +101,34 @@ export default function CollectibleDetails({ collectible }) {
           />
         }
       />
-      <Box padding={[4, 2, 4, 2]}>
-        <div className="collectible-details">
-          <Box margin={3} padding={2} justifyContent={JUSTIFY_CONTENT.CENTER}>
+      <Box className="collectible-details">
+        <div className="collectible-details__top-section">
+          <Card
+            padding={0}
+            justifyContent={JUSTIFY_CONTENT.CENTER}
+            className="collectible-details__card"
+          >
             <img
               className="collectible-details__image"
               src={collectibleImageURL}
             />
-          </Box>
+          </Card>
           <Box
-            margin={3}
             flexDirection={FLEX_DIRECTION.COLUMN}
-            width={
-              getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
-                ? BLOCK_SIZES.THREE_FOURTHS
-                : BLOCK_SIZES.HALF
-            }
+            className="collectible-details__top-section__info"
```

Comment:
I don't have a great solution for this but this width prop 
```
width={
              getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
                ? BLOCK_SIZES.THREE_FOURTHS
                : BLOCK_SIZES.HALF
            }
 ```
 was preventing this:
<img width="951" alt="Screen Shot 2021-12-09 at 9 57 41 PM" src="https://user-images.githubusercontent.com/34557516/145514971-58ab4bdd-162e-4a33-ada2-49e9ffbfd1e1.png">
 
 What do you recommend here?

Reply:
Ahhh ok. Hmm I would just truncate long ids? Would providing a copy to clipboard function add any value to the user?

Reply:
Hmm in the mean time we could just make it wrap have pushed a fix.